test/e2e/capi_test.go

```diff
@@ -265,14 +242,15 @@ var _ = Describe("Running the Cluster API E2E tests", func() {
 						InitWithBootstrapProviders:      []string{"kubeadm:" + e2eConfig.GetVariable(OldCAPIUpgradeVersion)},
 						InitWithControlPlaneProviders:   []string{"kubeadm:" + e2eConfig.GetVariable(OldCAPIUpgradeVersion)},
 						InitWithInfrastructureProviders: []string{"azure:" + e2eConfig.GetVariable(OldProviderUpgradeVersion)},
+						InitWithAddonProviders:          []string{"azure:" + "v0.1.1-alpha.0"},
```

Comment:
TODO: add env var once we can make sure this version works since v0.1.1-alpha.0 is set to CAPI v1.5.0 not 1.5.4.